.circleci/config.yml

```diff
@@ -67,50 +53,27 @@ jobs:
       - prodsec/security_scans:
           mode: auto
 
-  # This deploys a dev version
-  deploy-to-dev:
+  deploy-dev:
     <<: *defaults
     steps:
       - checkout
-      - attach_workspace:
```

Comment:
The jobs seems to rely on files created within these jobs, so the workspace attachment and restoration seems to waste more time than it gains.